style.css

```diff
@@ -1,825 +1,810 @@
-    margin: 4px 31px;
-  }
-
-  .header__menu {
-    height: 54px;
-    width: 54px;
+    grid-template-columns: 32% 68%;
+    grid-template-rows: 900px;
   }
 
-  .header__menu-item {
-    height: 3px;
-    width: 22.7px;
-    margin-bottom: 4px;
+  .first__sidebar {
+    grid-column: 1;
+    background: black;
   }
 
-  .header__language {
-    left: 29.5%;
-    top: 24px;
-    font-size: 12px;
+  .first__sidebar-social {
+    display: flex;
+    justify-content: space-between;
+    width: 155px;
+    padding: 816px 51px 0;
   }
 
-  .header__vector {
-    left: 32%;
-    top: 31px;
-    width: 9px;
-    height: 6px;
+  .first__sidebar-social-vk,
+  .first__sidebar-social-fb {
+    margin-left: 12px;
   }
 
-  .img__img {
-    width: 329px;
-    height: 338px;
+  .first__collection {
+	  background: #fff;
   }
 
   .step {
     position: absolute;
     width: 100%;
     display: grid;
-    grid-template-columns: 329px 329px;
-    grid-template-rows: 338px;
+    grid-template-columns: 439px 439px;
+    grid-template-rows: 454px;
     justify-content: center;
-    top: 101px;
+    top: 168px;
     z-index: 2;
-    grid-column-gap: 31px;
+    grid-column-gap: 54px;
   }
 
-  .text {
-    width: 329px;
-    height: 338px;
+  .img__img {
+    height: 450px;
   }
 
-  .text__title {
-    font-size: 49px;
-    line-height: 50px;
-    letter-spacing: 1px;
-    width: 329px;
+  .text {
+    height: 450px;
   }
 
   .text__container {
-    padding: 27px 0;
+    padding: 80px 0;
   }
 
-  .text__subtitle {
-    font-size: 9px;
-    line-height: 13px;
-    letter-spacing: 1px;
-    margin-bottom: 24px;
+  .text__title {
+    font-size: 60px;
+    line-height: 68px;
+    letter-spacing: 4px;
+    width: 439px;
   }
 
-  .collection__next-text {
-    font-size: 8px;
-    padding: 7px 22px;
-    margin-right: 31px;
-    letter-spacing: 0;
+  .text__subtitle {
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 1.1px;
+    margin-bottom: 40px;
   }
 
-  .collection__other-text {
-    font-size: 8px;
-    letter-spacing: 0;
-    margin-right: 3px;
-  }
+.collection__next-text {
+  text-decoration: none;
+  color: black;
+  font-style: normal;
+  font-size: 13px;
+  border: 1px solid black;
+  padding: 10px 23px;
+  margin-right: 31px;
+}
 
-  .collection__vector {
-    width: 27px;
-  }
+.collection__other-text {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 13px;
+  letter-spacing: 0;
+  margin-right: 14px;
+}
 
-  .collection__scroll {
-    width: 17px;
-    height: 28px;
-    left: 46.5%;
-    top: 489px;
-  }
+.collection__scroll {
+  width: 27px;
+  height: 46px;
+  margin-left: 25%;
+  margin-top: 815px;
+  border: 2px solid #000;
+  box-sizing: border-box;
+  border-radius: 15px;
+}
 
-  .collection__scroll-elem {
-    left: 47.3%;
-    top: 494.5px;
-    width: 2px;
-  }
+.collection__scroll-elem {
+  width: 2.45px;
+  height: 8.21px;
+  margin-left: 50%;
+  margin-top: 8px;
+  background: #000;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)